.eslintrc

```diff
@@ -252,7 +254,8 @@
         "MemberExpression": 1,
         "SwitchCase": 1,
         "ignoredNodes": [
-          "ConditionalExpression"
+          "ConditionalExpression",
+          "ClassDeclaration"
```

Comment:
Why is this ignored?

Reply:
I added this so that we can do this:

![image](https://user-images.githubusercontent.com/10142938/147930735-656bd6aa-d323-4ddc-ac89-779dc1a83371.png)

Otherwise, we are forced to have the same number of spaces before `extends` and before `blogPostData` making it confusing to read.
![image](https://user-images.githubusercontent.com/10142938/147930770-b9a8832a-b83e-443e-a372-e31b5f0e60c6.png)

Reply:
So will this change allow developers to add an arbitrary amount of spaces for the class definition?

Reply:
The arbitrary amount is allowed only for the second line, the first line will get reported if it is wrongly indented.

Reply:
That's weird, let's not lock this PR but I would suggest creating a GitHub-issue for fixing this issue (maybe create our own lint check for this)?

Reply:
Done.